src/webui/api/synccontroller.cpp

```diff
@@ -369,13 +371,26 @@ namespace
 //  - "up_rate_limit: upload speed limit
 //  - "queueing": priority system usage flag
 //  - "refresh_interval": torrents table refresh interval
+//  - "free_space_on_disk": Free space on the default save path
 // GET param:
 //   - rid (int): last response id
 void SyncController::maindataAction()
 {
     auto lastResponse = sessionManager()->session()->getData(QLatin1String("syncMainDataLastResponse")).toMap();
     auto lastAcceptedResponse = sessionManager()->session()->getData(QLatin1String("syncMainDataLastAcceptedResponse")).toMap();
 
+    if (m_freeDiskSpaceThread == nullptr) {
```

Comment:
nullptr_t is implicitly convertible to bool so we usually write it shorter: `if (m_freeDiskSpaceThread)`.